package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "axios": "^0.21.1",
     "body-parser": "^1.19.0",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v2.0.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-multiple-fallbacks",
```

Comment:
Reminder to self to upgrade after https://github.com/bravetechnologycoop/brave-alert-lib/blob/main/CHANGELOG.md has been merged.

Reply:
Done.